dimelo/load_processed.py

```diff
@@ -14,16 +14,28 @@ def pileup_counts_from_bedmethyl(
     bedmethyl_file: str | Path,
     motif: str,
     regions: str | Path | list[str | Path] | None = None,
+    # window_size: int | None = None,
+    single_strand: bool = False,
 ) -> tuple[int, int]:
     """
-    Extract number of modified bases and total number of bases from the given bedmethyl file
+    Extract number of modified bases and total number of bases from the given bedmethyl file.
+    Called by plotters or by the user.
 
-    TODO: How to name this method?
+    This function loops through all the provided regions and pulls those regions up in your
+    sorted and indexed bedmethyl file. For rows within those regions, we check that the motif
+    is correct (i.e. sequence context, modified base, mod code, and optionally strand). All
+    correct locations are included in the sum counts that get returned.
+
+    If no regions are specified, we return the sum total for the motif of interest across the
+    entire bedmethyl file.
 
     Args:
         bedmethyl_file: Path to bedmethyl file
         regions: Path to bed file specifying regions
         motif: type of modification to extract data for
+        window_size: (currently disabled) window around center of region, +-window_size//2
```

Comment:
Why is window size currently added and disabled?

Reply:
Because it breaks tests (they still run, but the way the tests are set up in dimelo_text.py, when running pileup_counts_from_bedmethyl the tester will see if there is a window_size parameter on the function and pass it if there is. This means that if we add one, the return changes). That is fine moving forward, but I wanted to merge in these changes without having to change the test results, then confirm that everything makes sense when running the package beyond just the tests, and if it does, we can make the simple change of re-enabling window_size for counts and then re-generate the test results reference data so that the tests pass with the change.